vscode/microsoft-kiota/src/extension.ts

```diff
@@ -60,18 +61,22 @@ export async function activate(
   await loadTreeView(context);
   await checkForLockFileAndPrompt(context);
   let codeLensProvider = new CodeLensProvider();
+  let deepLinkParams: Record<string, string|undefined> = {};
   context.subscriptions.push(
     vscode.window.registerUriHandler({
       handleUri: async (uri: vscode.Uri) => {
         if (uri.path === "/") {
           return;
         }
+        console.log(uri);
```

Comment:
This is likely unintentionally left in here

Reply:
Not really. Left there to help with debugging.